src/macros.h

```diff
@@ -278,10 +278,6 @@
      static inline int isnan(double x)
 	{ return x != x; }
 #   endif
-#   if defined(VMS) && defined(X86_64)
-     static inline int isinf(double x)
-        { return !isnan(x) && !finite(x); }
-#   endif
```

Comment:
Not needed anymore? Does this relates to `__CRTL_VER_OVERRIDE`?

Reply:
Yes, this was needed because the compiler was limited to a certain version by __CRTL_VER_OVERRIDE.
Now, when this limitation is removed, all works fine.